### PR TITLE
Signup: Fix logo SVG on mobile devices

### PR DESCRIPTION
**Pull request by sixhours**
#### Changes proposed in this Pull Request

* Downsize the initial W logo and upsize with transition rather than starting small and upsizing. The latter method caused a pixelated logo on mobile screens.

**Before:**

img_6140

**After:**

img_6141

#### Testing instructions

* Switch to this PR and navigate to `/start` on your mobile device
* Sign up for a free account; note the animation on the transition from the Plans page to the Checklist. Hopefully it's not blurry. :)

### CODE REVIEW

File: client/signup/header/style.scss [hunk 1]
```diff
@@ -2,45 +2,53 @@
 // the signup flow.
 .header {
 	position: absolute;
-		top: 0;
-		right: 0;
-		left: 0;
+	top: 0;
+	right: 0;
+	left: 0;
```
Comment (shaunandrews) [inline client/signup/header/style.scss]: Our code guidelines actually say we're supposed to indent these values.

File: client/signup/header/style.scss [hunk 2]
```diff
@@ -2,45 +2,53 @@
...
-			transform: scale( 10 ) translateY( 20px );
+			transform: scale( 1 ) translateY( 60px );
 		}
 	}
 
 	.header__left {
 		position: absolute;
-			top: 12px;
-			left: 16px;
+		top: 12px;
```
Comment (shaunandrews) [inline client/signup/header/style.scss]: Same as above.

File: client/signup/header/style.scss [hunk 3]
```diff
@@ -2,45 +2,53 @@
...
+		left: 16px;
 	}
 
 	.header__right {
 		position: absolute;
-			top: 12px;
-			right: 16px;
+		top: 12px;
```
Comment (shaunandrews) [inline client/signup/header/style.scss]: Same as above.

File: client/signup/header/style.scss (hunk 1 above)
Comment (sixhours) [reply client/signup/header/style.scss]: Looks like the linter or something in Github is automatically removing the extra indents; I just re-added and committed them, but they're not taking.

Comment (shaunandrews) [reply client/signup/header/style.scss]: Strange. Not a blocker by any means.

**Comment by shaunandrews** [comment]
🚢

**Pull request merged**